docs/specs/moniker.yml

```diff
@@ -1428,8 +1428,10 @@ inputs:
     }
 outputs:
   .errors.log: |
-    ["error","moniker-range-invalid","Invalid moniker range: 'netcore-1.2': Moniker `netcore-1.2` is not defined","docs/v1/a.md",2,15]
-    ["error","moniker-range-invalid","Invalid moniker range: '<= netcore-1.2': Moniker `netcore-1.2` is not defined","docs/v1/b.md",2,1]
+    ["error","moniker-range-invalid","Invalid moniker range 'netcore-1.2': Moniker 'netcore-1.2' is not defined","docs/v1/a.md",2,15]
+    ["error","moniker-range-invalid","Invalid moniker range '<= netcore-1.2': Moniker 'netcore-1.2' is not defined","docs/v1/b.md",2,1]
+    ["error","moniker-range-out-of-scope","No moniker intersection between docfx.yml/docfx.json and file metadata. Config moniker range '>= netcore-1.0' is 'netcore-1.0', while file moniker range 'netcore-1.2' is ","docs/v1/a.md",2,15]
```

Comment:
>is  [](start = 210, length = 3)

is empty?

Reply:
would it be enough to add `''` to become `while file moniker range 'netcore-1.2' is ''`?

Reply:
yeah, that would also be OK

---
In reply to: [404512881](https://github.com/dotnet/docfx/pull/5704#discussion_r404512881) [](ancestors = 404512881)